pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -444,6 +445,24 @@ private Long configureRefreshDelay(final Long refreshDelay) {
   )
   private final Collection<String> accountsWhitelist = null;
 
+  @Option(
+    names = {"--privacy-url"},
+    description = "The URL on which enclave is running "
+  )
+  private final URI privacyUrl = PrivacyParameters.DEFAULT_ORION_URI;
```

Comment:
may `DEFAULT_ORION_URI` be `DEFAULT_ORION_URL` if the content is a URL (and URL is a URI, hence the type, but semantically if you ask the user for a URL, the default value should be called a URL too IMO)